### DISCUSSION TITLE
Thought this was an interesting comparison(GPU speed)

### DISCUSSION THREAD

**Discussion by Watcher2747** [mrdbourke/pytorch-deep-learning#278] [General]
While following the video:
https://youtu.be/Z_ikDlimN6A?t=67273

I noticed that the time of training time on my **free** GPU is quite lower than the one that is supposed to be for PRO users.

### Benchmark from the video:
![image](https://user-images.githubusercontent.com/49002577/214792318-f7cd8537-fcd8-4607-8ecb-82466b05d3aa.png)

### Benchmark from my notebook:
![image](https://user-images.githubusercontent.com/49002577/214792528-a84f9573-780d-4486-8c89-b401a77613b0.png)

I assume that this is simply due to the passage of time and that both sides got upgraded GPUs after the video was made, but I still thought this was interesting enough to share.

_or who knows, maybe that is just the power of dark mode_

**Comment by Watcher2747** [comment]
### hah, continued watching and:
![image](https://user-images.githubusercontent.com/49002577/214794300-faeb3457-b950-46a8-8dfa-7901508f0180.png)

_mine is still faster for model_0_

**Comment by mrdbourke** [comment]
Hey @Watcher2747 ,

Good observation.

Yes, you're right the GPU hardware running Colab can vary quite often nowadays.

My guess is: new GPUs are being used and old GPUs are being swapped/replaced.

Or... it could be *dark mode*! 🌚

Good news though because faster is generally better!
